pkg/cloud/models/models.go

```diff
@@ -273,7 +273,7 @@ type Provider interface {
 	GetAddresses() ([]byte, error)
 	GetDisks() ([]byte, error)
 	GetOrphanedResources() ([]OrphanedResource, error)
-	NodePricing(Key) (*Node, error)
+	NodePricing(Key) (*Node, PricingMetadata, error)
```

Comment:
This change just allows us to use the very same function for getting pricing, and reporting metadata about where that data came from.